erpnext_com/www/docs/user/manual/en/CRM/lead.md

```diff
@@ -74,7 +74,7 @@ You can add the details of all such people in the same lead. Once you save a Lea
 ### 2.3 Recording Comments, Emails and Events
 
 * **Comments**: You can write your comment in 'Add a comment' box and click on 'Comment'.
-* **Emails**: You can send an email to the lead by clicking on the 'New Email' button and when the lead responds to your email, it will visible.
+* **Emails**: You can send an email to the lead by clicking on the 'New Email' button and when the lead responds to your email, it will be visible.
```

Comment:
You can send an email to the lead by clicking on the 'New Email' button and when the lead responds to your email, it will appended to your email creating an email thread.